### PR TITLE
[3.6] Backport README.rst changes

### PR DESCRIPTION
**Pull request by Mariatta**
Backport of the following PRs:
GH-104
GH-145
GH-55
GH-283
GH-295

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,6 +15,22 @@ reserved.
...
 
+General Information
+-------------------
+
+- Website: https://www.python.org
+- Source code: https://github.com/python/cpython
+- Issue tracker: https://bugs.python.org
+- Documentation: https://docs.python.org
+- Developer's Guide: https://cpython-devguide.readthedocs.io
+
```
Comment (ned-deily) [inline README.rst]: The readthedocs URL is proposed here but the docs.python.org is proposed in the changes for Doc/README.rst and further down in this file.  I say stick with the latter.

**Review by ned-deily — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (Mariatta) [reply README.rst]: Thanks, @ned-deily :)
I changed the url on the `master` branch in GH-295. Once that is merged, I'll cherry-pick that to this PR. Will that work?

Comment (ned-deily) [reply README.rst]: Sure.  Just do it!

Comment (Mariatta) [reply README.rst]: Done!

**Review by ned-deily — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks for the review @ned-deily 😸  🎉

**Pull request merged**